nntrainer/layers/lstm.cpp

```diff
@@ -189,23 +189,11 @@ void LSTMLayer::forwarding(RunLayerContext &context, bool training) {
   const float dropout_rate = std::get<props::DropOutRate>(lstm_props).get();
   const unsigned int max_timestep =
     std::get<props::MaxTimestep>(lstm_props).get();
-  const props::Timestep timestep = std::get<props::Timestep>(lstm_props);
```

Comment:
Is there any reason to remove the time step? That means the height of input is always 1?

Reply:
No. The height of input can be greater than 1.
Timestep property was used to unrolling the lstm layer like lstmcell layer. 
AFAIK this property is introduced because we did not implemented lstmcell layer but now we have so I removed this property.